components/psst/browser/core/BUILD.gn

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+component("core") {
+  output_name = "psst_browser_core"
+
+  defines = [ "IS_PSST_BROWSER_CORE_IMPL" ]
+
+  sources = [
+    "psst_component_installer.cc",
+    "psst_component_installer.h",
+    "psst_rule.cc",
+    "psst_rule.h",
+    "psst_rule_service.cc",
+    "psst_rule_service.h",
+  ]
+
+  deps = [
+    "//base",
+    "//brave/components/brave_component_updater/browser",
+    "//brave/components/psst/common",
+    "//components/component_updater",
+    "//crypto",
+    "//extensions/common",
```

Comment:
you need to use `//brave/extensions/common` here or it won't work on ios or android

Reply:
https://sourcegraph.com/github.com/brave/brave-core/-/blob/extensions/BUILD.gn?L8